docker-compose.yml.template

```diff
@@ -14,8 +14,7 @@ services:
     environment:
       - QLF_API_URL=http://localhost:8001/dashboard/api
       - QLF_BASE_URL=http://localhost:8001
-      - BOKEH_SERVER=loclahost
-      - QLF_DAEMON_HOST= localhost
```

Comment:
QLF DAEMON HOST is also gone?

Reply:
@ricardogando yes, that was what broke it actually...